packages/language-support/src/types.ts

```diff
@@ -34,4 +36,9 @@ export type Neo4jFunction = {
   returnDescription: string;
   signature: string;
   aggregating: boolean;
+  isDeprecated: boolean;
+};
+
+export type CompletionItem = VSCodeCompletionItem & {
+  signature?: string;
 };
```

Comment:
The reason for overriding this type is that I need to copy the signature into the `details` of `CompletionItem` after the type  (e.g. `(function)`) for VSCode to show it in the right place. But for codemirror we need the type in a different place than the method information.